packages/taquito-rpc/src/types.ts

```diff
@@ -11,6 +11,71 @@ export type DelegateResponse = string | null;
 
 export type OperationHash = string;
 
+interface Before {
+  value?: string;
+  node?: string;
+}
+
+interface After {
+  value?: string;
+  node?: string;
+}
+
+interface State {
+  inode?: Inode;
+  other_elts?: any;
+  inode_extender?: InodeExtender;
+}
+
+interface Inode {
+  length: number;
+  proofs?: string[];
```

Comment:
I understood the type as follow:
`proofs: [string | null, string | null]`
an array of 2 elements that can be a contest_hash or null